README.md

```diff
@@ -267,16 +267,18 @@ twitch_miner.mine(
         "streamer-username10",
         "streamer-username11"
     ],                                 # Array of streamers (order = priority)
-    followers=False                    # Automatic download the list of your followers (unable to set custom settings for you followers list)
+    followers=False,                   # Automatic download the list of your followers (unable to set custom settings for you followers list)
+    order=FollowersOrder.ASC           # Order of the followers list. ASC or DESC
```

Comment:
Can probably precise what order it refers to. I didn't quite check but I think it's sorted by follow date.

Reply:
Yup, tested and It's by follow date